packages/backend/src/open_payments/receiver/service.ts

```diff
@@ -97,13 +99,14 @@ async function createLocalIncomingPayment(
   args: CreateReceiverArgs,
   walletAddress: WalletAddress
 ): Promise<OpenPaymentsIncomingPaymentWithPaymentMethods | ReceiverError> {
-  const { expiresAt, incomingAmount, metadata } = args
+  const { expiresAt, incomingAmount, metadata, tenantId } = args
 
   const incomingPaymentOrError = await deps.incomingPaymentService.create({
     walletAddressId: walletAddress.id,
     expiresAt,
     incomingAmount,
-    metadata
+    metadata,
+    tenantId: tenantId ?? Config.operatorTenantId
```

Comment:
Instead of using `Config.operatorTenantId` we should just throw if `tenantId` is undefined, since its the responsibility of the caller of the function (in this case, the Admin API resolver) to provide tenantId if its a local payment.

This means we should add `tenantId` to the `createReceiver` mutation